### PR TITLE
Focus style updating

### PR DESCRIPTION
**Pull request by flacoman91**
Updating the Focus view more link to add a background and right align the text.
updating the demos

Screen Shot 2020-08-04 at 1 10 09 PM
Screen Shot 2020-08-04 at 1 10 04 PM

### CODE REVIEW

File: test/specs/row.spec.js [hunk 1]
```diff
@@ -698,6 +683,46 @@ define(['d3', 'row', 'rowChartDataBuilder'], function(d3, chart, dataBuilder) {
...
+        let rowChart, dataset, containerFixture, f;
+
+        beforeEach(() => {
+            dataset = buildDataSet('withSeparatorsNoDelta');
+            rowChart = chart().enableLabels(true);
+
+            // DOM Fixture Setup
+            f = jasmine.getFixtures();
+            f.fixturesPath = 'base/test/fixtures/';
+            f.load('testContainer.html');
```
Comment (JeffreyMFarley) [inline test/specs/row.spec.js:696]: 😎

File: dist/css/charts/row.css [hunk 2]
```diff
@@ -22,3 +22,10 @@
 
 .row-chart .percentage-label {
   fill: #666A73; }
+
+.row-chart .view-more-group .view-more-background {
+  fill: aqua; }
```
Comment (JeffreyMFarley) [inline dist/css/charts/row.css:27]: This is able to be overwritten correct?

Comment (flacoman91) [reply dist/css/charts/row.css:27]: yes, correct. that's only if someone decides to import the css.  I had to have that in here so the demo would work.

**Review by JeffreyMFarley — APPROVED** [review]
Let'r rip

**Pull request merged**